bundler/lib/bundler/man/bundle-viz.1

```diff
@@ -15,6 +15,9 @@
 .P
 The associated gems must also be installed via \fBbundle install(1)\fR \fIbundle\-install\.1\.html\fR\.
 .
+.P
+\fBviz\fR command was deprecated in Bundler 2\.2\. Use bundler\-graph plugin \fIhttps://github\.com/rubygems/bundler\-graph\fR instead\.
+.
```

Comment:
This seems unrelated, problem with rebase maybe?

Reply:
Not entirely sure what happened there either.
I think it may have to do with me having had merged master
into this branch earlier but then later rebasing it.

After another rebase it is gone now. Sorry for the noise 🤷

Reply:
No worries, thanks!